deploy/helm-chart/README.md

```diff
@@ -105,6 +124,53 @@ the release `my-release`, run:
 helm upgrade my-release . -n nginx-gateway
 ```
 
+### Configure Delayed Termination for Zero Downtime Upgrades
+
+To achieve zero downtime upgrades (meaning clients will not see any interruption in traffic while a rolling upgrade is
```

Comment:
I wonder if we should include more details on why and in what environment the delayed termination is needed. Is this a commonly known problem? @pleshakov

Reply:
I think there might be a couple of different factors (e.g. could be external load balancer taking time to update with the new pod IPs so still sending traffic to terminating pods), and I'm not sure exactly of all reasons why this might be necessary. Do we think this is a required expiation for our docs?

Reply:
Fair enough. I think it's fine to leave it out. We can always add it later if needed!